data/sles4sap/qe_sap_deployment/qesap_azure.yaml

```diff
@@ -9,6 +9,8 @@ terraform:
     hana_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
     iscsi_remote_python: "%ANSIBLE_REMOTE_PYTHON%"
     hana_instancetype: "%HANA_INSTANCE_TYPE%"
+    vnet_address_range: "%VNET_ADDRESS_RANGE%"
+    subnet_address_range: "%SUBNET_ADDRESS_RANGE%"
```

Comment:
LGTM, my only question is, do we want `vnet_address_range` and `subnet_address_range` vars defined in the main yaml, or we want a new yaml schedule specifically for maintenance jobs that will include them? 

I'm asking in case we want to have runs using `qesap_azure_yaml` that won't need the mirror, and thus won't need peering and address ranges as well